users-backend/users/domain/entities.py

```diff
@@ -91,6 +112,13 @@ def display_name(self) -> str:
     Column("is_active", Boolean(), default=True, nullable=False),
     Column("is_staff", Boolean(), default=False, nullable=False),
     Column("is_superuser", Boolean(), default=False, nullable=False),
+    Column(
+        "request_reset_password_id",
```

Comment:
I'd make this a bit more generic as we can use also to invalidate existing login tokens (when someone logs out from everywhere, when they change password, when we ban them)

Reply:
sure

Reply:
consider that every time the user sends a request to our gateway we get information about the user, so it's easier for us invalidating tokens and so on